### PR TITLE
Adds a new atmos gas: Slime Pheromones

### PR DESCRIPTION
**Pull request by BurgerLUA**
[Changelogs]:
:cl: BurgerBB
add: Adds a new atmos gas, Slime Pheromone, which increases the breathers arousal levels when inhaled. They can be produced naturally by slimes, or can be found in canisters hidden away in maintenance.
balance: Slimes now produce Pheromones when they are feeding on something, or are in the process of moving to feed on something. If there are too many pheromones in a room (more than 10% of the air is pheromones), slimes become rabid.
/:cl:

[why]: I ask myself this on a daily basis but honestly this is a xenobio nerf. About an hour into xenobio you're probably going to want to clean the cells via portable scrubbers if you have too many slimes in a single pen. It's actually insane how many slimes you can stuff into one cell, and this nerfs this.

Also boner stuff is always good for citadel.

### CODE REVIEW

**Comment by CalamaBanana** [comment]
Always a fan of more gases and more "citadel" stuff.

- It should trigger air alarms, currently you have set them to ignore it. While not directly harmful, it still is an impurity impacting normal working procedure, maybe use the same values as for Miasma there.
- I would also suggest a (low, maybe 1%) minimum concentration of it being present before it affects players. The Xenobiologist just needs to clumsily open his pens once to leak out trace amounts over half the station, would get old very fast.


Random thoughts: 
- Make it usable to create slimes. Water vapor is dangerously unused, have those two gases react when in high concentration and use a large amount of their moles to form a slime body. If you are feeling creative, make multiple recipes, water+pheromones+plasma for purple slimes for example.
- Have it create slime bodies when ran through the SM. (Which do end up sucked into the SM, supercharging it)

File: modular_citadel/code/modules/reagents/reagents/cit_reagents.dm [hunk 1]
```diff
@@ -164,6 +164,25 @@
...
 	..()
 
+/datum/reagent/drug/pheromones
+	name = "slime pheromones"
+	id = "pheromones"
+	description = "Pheromones released by slimes, in liquid concentrated form. Has about the same strength of Hexacrocin, but without the brain damage and addiction."
+	taste_description = "ants"
+	color = "#FFADFF"
+
+/datum/reagent/drug/pheromones/on_mob_life(mob/living/M)
```
Comment (Ghommie) [inline modular_citadel/code/modules/reagents/reagents/cit_reagents.dm:175]: Copypasta minus moaning and trait immunity check, make this a subtype of aphrodisiacplus with the addiction and overdose values set to 0. If you want to retain the no emoting and immunity bypass, just make a couple additional vars for them.

File: code/modules/mob/living/simple_animal/slime/life.dm [hunk 2]
```diff
@@ -147,6 +155,15 @@
 			stat = CONSCIOUS
 			update_canmove()
 			regenerate_icons()
+		else if(stat == CONSCIOUS && frenzy && !rabid && !stasis)
+			to_chat(src, "<span class='danger'>The amount of pheromones in the air is driving you mad!</span>")
+			rabid = 1
```
Comment (Ghommie) [inline code/modules/mob/living/simple_animal/slime/life.dm:160]: TRUE this.

**Comment by deathride58** [comment]
I'm really not sure what to feel about having the pheromone canisters mapped in roundstart, especially in an easy-to-access area like maint.

**Comment by chaoko99** [comment]
I can't wait to work with Xenobio to get this into distro.

**Comment by BurgerLUA** [comment]
> I'm really not sure what to feel about having the pheromone canisters mapped in roundstart, especially in an easy-to-access area like maint.

I'll revert the map changes then.

**Comment by silicons** [comment]
i'm really not sure what to feel about the latest spike in memey erp-ish feature prs rather than gameplay features

and the fact that linda performance goes down when you get more gases in the air and this is going to be spammed. hard.

**Comment by Ghommie** [comment]
> and the fact that linda performance goes down when you get more gases in the air and this is going to be spammed. hard.

At this point, this is better closed. It's going to be more of a nuisance than something good. Aphro and Aphro+ already exist.

**Comment by BurgerLUA** [comment]
> > and the fact that linda performance goes down when you get more gases in the air and this is going to be spammed. hard.
> 
> At this point, this is better closed. It's going to be more of a nuisance than something good. Aphro and Aphro+ already exist.

tbh I find it a little disingenuous that you tone changed from "here are some tweaks you can do" to "close it now" after I disagreed with you on discord over an argument

this pr is a bit of a meme but it's still a gameplay feature that antags can use to make better use of xenobiology, as well as a small nerf to xenobiology that punishes people who leaves too many starving slimes in their cells for too long. I played xenobiology for a bit and I noticed that there is 0 risk of min/maxing the amount of slimes in a cell, except for the very minimal miasma gas that spawns if you don't clean up the monkeys, but it's very insignificant.

It's a meme with a purpose, and that purpose is a first step towards making xenobiology relevant to antags, that isn't just gold core slime spam.

**Comment by BurgerLUA** [comment]
I reworked the pheromone reagent to be more slime-like and not just a crocin ripoff.

**Comment by Ghommie** [comment]
I don't recall having any argument related to this on Discord. But yea, sorry for letting a personal feeling toward this PR take over after a potential performance drag was cited.

**Comment by silicons** [comment]
i can't speak on linda performance with just one gas addition for sure but honestly i'm not very convinced this is worth it.

**Comment by BurgerLUA** [comment]
> i can't speak on linda performance with just one gas addition for sure but honestly i'm not very convinced this is worth it.

Citadel has no custom gases and a lot of people are eager for this change as they are suggesting plenty of things you can create and do with the gas, other than the boner medication. More PRs will probably be done on this, and I think that integrating atmos mechanics in xenobio would be interesting for it.

**Comment by silicons** [comment]
sorry i don't consider this worth it either for the crocin effect nor for the slime thing, there's better ways to handle xenobio doing the open slime farm meme than to make it a gas that people will inevitably abuse + the overhead from it.

**Pull request closed without merging**